content/en/contribute/code/releasing/_index.md

```diff
@@ -44,6 +44,17 @@ Given a version number `MAJOR.MINOR.PATCH`, increment the:
 {{% alert title="Info" %}} You can find the versions currently supported, dependencies, and release notes for the CHT Core 
  [on the Releases page]({{% ref "core/releases" %}}). {{% /alert %}}
 
+### When to wait to release
+
+In general, we do not want to wait to do a release.  The moment the release manager feels the release is ready, they should release it.  This may even be with just one ticket! 
+
+There are two conditions that we would wait:
+
+1. There is a new bug found in `master` that has not been released.  This should be fixed before the release.
+1. There is a bug found in an existing release that can be fixed in 5 days.  In less than 5 days the fixed code will be released.  In more than 5 days, the bug will be released and a new release will ship later with the fix for the bug.
+
+The reason we have these rules around when we wait, is so that everyone knows what they are and the release manager never needs to ask "should we wait?" - they'll know already!
```

Comment:
```suggestion
The reason we have these rules around when we wait, is so that everyone knows what they are and the release manager never needs to ask "should we wait?" - they'll know already!

While waiting the release manager should use the time effectively by doing as many release steps as possible including writing release notes, preparing the release branch, etc so that the release can be finalized as quickly as possible once the fixes are made.
```

Reply:
Same here, please replace the mention of `we` with "release manager" or something more generic.

Reply:
took both in 35a099a8  - thanks!